README.md

```diff
@@ -44,52 +44,47 @@
 ```
 
+<sub>**Note:** If the environment variable `$XDG_CONFIG_HOME` is present, it will place the `nvm` files there.</sub>
+
 **Note:** You can add `--no-use` to the end of the above script (...`nvm.sh --no-use`) to postpone using `nvm` until you manually [`use`](#usage) it.
 
 You can customize the install source, directory, profile, and version using the `NVM_SOURCE`, `NVM_DIR`, `PROFILE`, and `NODE_VERSION` variables.
 Eg: `curl ... | NVM_DIR="path/to/nvm"`. Ensure that the `NVM_DIR` does not contain a trailing slash.
 
 <sub>*NB. The installer can use `git`, `curl`, or `wget` to download `nvm`, whatever is available.*</sub>
 
-**Note:** On Linux, after running the install script, if you get `nvm: command not found` or see no feedback from your terminal after you type:
-
-```sh
-command -v nvm
-```
-
-simply close your current terminal, open a new terminal, and try verifying again.
+**Note:** On Linux, after running the install script, if you get `nvm: command not found` or see no feedback from your terminal after you type `sh command -v nvm
+`, simply close your current terminal, open a new terminal, and try verifying again.
 
 **Note:** Since OS X 10.9, `/usr/bin/git` has been preset by Xcode command line tools, which means we can't properly detect if Git is installed or not. You need to manually install the Xcode command line tools before running the install script, otherwise, it'll fail. (see [#1782](https://github.com/nvm-sh/nvm/issues/1782))
 
-**Note:** On OS X, if you get `nvm: command not found` after running the install script, one of the following might be the reason:-
-
-  - your system may not have a `.bash_profile` file where the command is set up. Create one with `touch ~/.bash_profile` and run the install script again
-  - you might need to restart your terminal instance. Try opening a new tab/window in your terminal and retry.
+**Note:** On OS X, if you get `nvm: command not found` after running the install script, one of the following might be the reason:
 
-If the above doesn't fix the problem, open your `.bash_profile` and add the following line of code:
+  - Your system may not have a `.bash_profile` file where the command is set up. Create one with `touch ~/.bash_profile` and run the install script again
+  - You might need to restart your terminal instance. Try opening a new tab/window in your terminal and retry.
 
-`source ~/.bashrc`
+If the above doesn't fix the problem, you may try the following:
+  - Open your `.bash_profile` (or `~/.zshrc`, `~/.profile`, or `~/.bashrc`) and add the following line of code: `source ~/<your_profile_file>`. E.g. `source ~/.bashrc` or `source ~/.zshrc`. 
+  - If the above don't work, try adding the snippet from the install script (the one in the snippet below) to your profile (`~/.bash_profile`, `~/.zshrc`, `~/.profile`, or `~/.bashrc`).
+```sh
+export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
```

Comment:
I would prefer not to repeat these lines. Can this link to the section in [here](https://github.com/nvm-sh/nvm#install--update-script) instead?

Reply:
Sure, sounds good. I removed the snippet and referred to and linked to the lines in the install and update section.